### PR TITLE
feat(NewOrderAhead/Order Status Screen): Layout for the order status screen.

### PR DESCRIPTION
**Pull request by dabralyogesh12**
Issues: fixes https://github.com/SleekOrg/tracker/issues/298
feat(NewOrderAhead/Order Status Screen): Layout for the order status screen.
Description: Added the layout for the order status screen. The back button integration is pending and will be done once the Order Confirmation screen PR is pushed.
Verification: Tested locally.

### CODE REVIEW

File: src/features/stall/StatusStepper.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  createStyles({
+    root: {
+      width: '100%',
+    },
+    stepperLable: {
+      fontWeight: 700,
+    },
+  })
+);
+function getSteps() {
```
Comment (teamSleek) [inline src/features/stall/StatusStepper.tsx:72]: match states from https://github.com/SleekOrg/SleekRepo/blob/master/user_app/src/components/NewOrderView.js#L107

It seems that you are missing completed_pr and cancelled

File: src/features/stall/OrderStatus.tsx [hunk 2]
```diff
@@ -1,111 +1,171 @@
...
+              <Typography roboto component="div" variant="body2">
+                Order # 602C52KP312806
+              </Typography>
+              <Typography
+                roboto
+                component="div"
+                variant="subtitle1"
+                style={{ color: '#6D5CFF' }}
+              >
+                Detailed receipt
```
Comment (teamSleek) [inline src/features/stall/OrderStatus.tsx:89]: let us center align heading as we discussed.

**Review by teamSleek — CHANGES_REQUESTED** [review]

File: src/features/stall/OrderStatus.tsx (hunk 2 above)
Comment (dabralyogesh12) [reply src/features/stall/OrderStatus.tsx:89]: Done, updated PR.

File: src/features/stall/StatusStepper.tsx (hunk 1 above)
Comment (dabralyogesh12) [reply src/features/stall/StatusStepper.tsx:72]: I have added the status strings.

File: src/features/stall/OrderStatus.tsx [hunk 3]
```diff
@@ -1,111 +1,171 @@
...
+          >
+            <Grid item className={classes.commonStyle}>
+              <Typography roboto component="div" variant="body2">
+                Order # 602C52KP312806
+              </Typography>
+              <Typography
+                roboto
+                component="div"
+                variant="subtitle1"
+                style={{ color: '#6D5CFF' }}
```
Comment (teamSleek) [inline src/features/stall/OrderStatus.tsx:87]: let us not hardcode color here but use it from the theme i.e. theme's primary color

Comment (teamSleek) [reply src/features/stall/OrderStatus.tsx:87]: here and everywhere else

File: src/features/stall/OrderStatus.tsx [hunk 4]
```diff
@@ -1,111 +1,171 @@
...
+              </Typography>
+            </Grid>
+            <Grid item xs={12} sm={12} className={classes.commonStyle}>
+              <Typography component="div" variant="h4">
+                Amount Paid
+              </Typography>
+              <Typography
+                roboto
+                variant="h3"
+                style={{ color: '#6D5CFF', marginTop: '15px' }}
```
Comment (teamSleek) [inline src/features/stall/OrderStatus.tsx:127]: ditto

File: src/features/stall/StatusStepper.tsx (hunk 1 above)
Comment (teamSleek) [reply src/features/stall/StatusStepper.tsx:72]: can't see it here. Also, delete the collected?
Screen Shot 2021-04-08 at 9 37 01 PM

File: src/features/stall/OrderStatus.tsx (hunk 2 above)
Comment (teamSleek) [reply src/features/stall/OrderStatus.tsx:89]: This doesn't seem like the mobile version we discussed?
Screen Shot 2021-04-08 at 9 39 26 PM

File: src/features/stall/OrderStatus.tsx (hunk 3 above)
Comment (teamSleek) [reply src/features/stall/OrderStatus.tsx:87]: We should strive to not repeat any styling that is already part of the theme.

**Review by teamSleek — CHANGES_REQUESTED** [review]

File: src/features/stall/StatusStepper.tsx (hunk 1 above)
Comment (dabralyogesh12) [reply src/features/stall/StatusStepper.tsx:72]: Done, updated PR.

File: src/features/stall/OrderStatus.tsx (hunk 2 above)
Comment (dabralyogesh12) [reply src/features/stall/OrderStatus.tsx:90]: Center aligned the Detailed receipt with respect to Order Id.

File: src/features/stall/OrderStatus.tsx (hunk 4 above)
Comment (dabralyogesh12) [reply src/features/stall/OrderStatus.tsx]: Done, updated PR.

**Pull request merged**